### PR TITLE
Better syntax highlighting

### PR DESCRIPTION
**Pull request by chillu**
Specifically, the old highlighter didn't support the "diff" format which was important for changelogs.
Changed the style from dark to light, since it's way too contrasty on a light background - makes it hard to scan both text and code.
And reduced the padding between code and content (mostly using the highlight style defaults)
Added a slight background shadow on code to visually offset the relatively light background color
Changed existing background color for `<pre>` to a slightly lighter shade, better contrast
Didn't create a custom *.ss style yet, too much work, and XML is good enough for now

Related changelog reformatting at https://github.com/silverstripe/silverstripe-framework/pull/7082

Preview

image

Diff in revised changelog (the main reason for this)

image

PHP styles (and SQL)

image

Template styles

/cc @clarkepaul does that styling work for you? We can tweak it further, I'm not very happy with the spacing overall, but for me it's an improvement over what we had before.

### CODE REVIEW

File: code/controllers/DocumentationViewer.php [hunk 1]
```diff
@@ -101,12 +101,17 @@ public function init()
...
-            Requirements::javascript('https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js');
-
-            Requirements::javascript(DOCSVIEWER_DIR .'/javascript/DocumentationViewer.js');
+            Requirements::combine_files(
+                'docs.js',
+                [
+                    THIRDPARTY_DIR .'/jquery/jquery.js',
+                    DOCSVIEWER_DIR .'/thirdparty/highlight/highlight.pack.js',
+                    DOCSVIEWER_DIR .'/javascript/DocumentationViewer.js',
+                ]
```
Comment (robbieaverill) [inline code/controllers/DocumentationViewer.php]: Can you use old school array declaration here to let the PHP 5.3 builds pass?

Comment (chillu) [inline code/controllers/DocumentationViewer.php]: Done

**Comment by robbieaverill** [comment]
@ingo @tractorcow please check with @nicolewilliams16 before deploying this to ss.org

**Comment by nicolewilliams16** [comment]
@chillu @tractorcow @robbieaverill happy with this, go ahead thanks :)

**Comment by SpiritLevel** [comment]
Sad to see the high contrast code blocks go; I preferred them to the new style. Sorry Ingo!

**Pull request merged**